src/main/java/org/example/DatabaseHelper.java

```diff
@@ -0,0 +1,66 @@
+package org.example;
+
+import java.sql.*;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Objects;
+
+/*
+In this database mySQL database with table name audit.
+The table contains following columns, path to the file, the word searched, the date time of the search
+with error and success messages
+ */
+public class DatabaseHelper {
+    String driverClass = "com.mysql.cj.jdbc.Driver";
+    String mySqlUrl = "jdbc:mysql://localhost:3306/Poc1";
+    String userName = "root";
+    String passwordOfDatabase = "DivyaSathyan022960";
+    String dateAndTimeFormat = "dd/MM/yyyy HH:mm:ss";
+    String createTable = "create table audit (PathOfTheFile varchar(100), WordSearched varchar(50), DateAndTime varchar(50), FinalResult varchar(50), NumberOfWordCount int, ErrorMessage varchar(100))";
+
+    /*
+      Storing data to database
+     */
+    public void storingDataToDataBase(String filepath, String WordSearch, String finalResult, int totalNumberOfWords, String errorMessage) throws SQLException {
+        Connection connectionToDataBase = null;
+        Statement statement = null;
+        String DateAndTime;
+        DateTimeFormatter dateAndTimeFormat = DateTimeFormatter.ofPattern(this.dateAndTimeFormat);
+        LocalDateTime now = LocalDateTime.now();
+        DateAndTime = dateAndTimeFormat.format(now);
+        try {
+            connectionToDataBase = DriverManager.getConnection(this.mySqlUrl, this.userName, this.passwordOfDatabase);
```

Comment:
Should'nt we have the syntax Class.forname before line no 32

Reply:
Updated.